### PR TITLE
fix system test for kubernetes integration for k8s v1.27.0

### PR DESCRIPTION
**Pull request by gsantoro**
## What does this PR do?

Fix failing system tests for the integration kubernetes when testing with kubernetes > v1.27.0.

It fails with the following error

```
[error] failed to initialize Lua VM in /etc/nginx/nginx.conf:47
```

I tried to just upgrade the version of container image `k8s.gcr.io/nginx-slim:0.8` to a newer tag but since I couldn't easily list the tags available (like I would do with docker hub) I couldn't really figure out the Dockerfile related to that image or what was wrong with that image. Questions on stackoverflow about this error seems to suggest that the official image of nginx doesn't come with Lua at all.

Also because of [announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/) I tried to replace the container registry from k8s.gcr.io to registry.k8s.io but I had a similar problem trying to figure out what was the latest version of nginx image at that container registry.

So the solution was to use an official nginx image from dockerhub. That was enough to fix the issue and make all system tests pass.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [x] all tests pass on Jenkins

## How to test this PR locally

- [x] replicate the Jenkins pipeline locally

## Related issues

-

### CODE REVIEW

**Comment by constanca-m** [comment]
Hey @jsoriano, does merging this commit overwrite the beta versions in Kubernetes package? Beta versions are in 1.40.0 versions as well.

**Comment by jsoriano** [comment]
> Hey @jsoriano, does merging this commit overwrite the beta versions in Kubernetes package? Beta versions are in 1.40.0 versions as well.

The 1.40.0 will be published and beta versions will remain published too. Search requests for GA versions will start including 1.40.0.

File: packages/kubernetes/data_stream/container_logs/manifest.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ streams:
         title: Dataset name
         description: >
           Set the name for your dataset. Changing the dataset will send the data to a different index. For more info look at [data_stream field](https://www.elastic.co/guide/en/ecs/master/ecs-data_stream.html).
+
```
Comment (MichaelKatsoulis) [inline packages/kubernetes/data_stream/container_logs/manifest.yml:28]: Why is this line added? Same for the next file

Comment (gsantoro) [reply packages/kubernetes/data_stream/container_logs/manifest.yml:28]: it's the output of `elastic-package format` or `elastic-package lint`

**Review by constanca-m — APPROVED** [review]
You will have to update the K8s package version again since 1.41 exists now.

**Comment by gsantoro** [comment]
> You will have to update the K8s package version again since 1.41 exists now.

Do you mean this [PR](https://github.com/elastic/integrations/pull/6332). It hasn't been merged yet and I haven't got a feedback yet on what's the right naming scheme for beta versions and what comes next. I left a comment at [here](https://github.com/elastic/integrations/pull/6332/files#r1206357078).

**Comment by constanca-m** [comment]
> > You will have to update the K8s package version again since 1.41 exists now.
> 
> Do you mean this [PR](https://github.com/elastic/integrations/pull/6332). It hasn't been merged yet and I haven't got a feedback yet on what's the right naming scheme for beta versions and what comes next. I left a comment at [here](https://github.com/elastic/integrations/pull/6332/files#r1206357078).

You're right, when I saw it was pushed I though it had been committed as well. Sorry, my mistake :)

File: packages/kubernetes/data_stream/container_logs/manifest.yml (hunk 1 above)
Comment (MichaelKatsoulis) [reply packages/kubernetes/data_stream/container_logs/manifest.yml:28]: strange. Because there is not empty line under all fields. Personally I wouldn't leave this line but it's your choice.
